docs/src/components/Tabs/consts.ts

```diff
@@ -2,7 +2,9 @@ import { mediaQueries } from "@kiwicom/orbit-components";
 
 export const TAB_HEIGHT = "64px";
 export const BORDER_RADIUS = "12px";
-// to prevent tab shadow from clipping
-export const SHADOW_PADDING_TOP = "16px";
-export const SHADOW_PADDING_RIGHT = "24px";
+// taken from the stronger shadow that appears when hovering stacked tabs
+// we use these values for padding to prevent that shadow from clipping
+export const SHADOW_TOP = "16px";
+export const SHADOW_LEFT = "24px";
+export const SHADOW_RIGHT = "24px";
```

Comment:
i guess instead of consts you can use tokens. `spaceMedium` and `spaceLarge`

Reply:
`boxShadowRaised`, which is used for tabs, doesn't use the spacing tokens, the value is instead hardcoded, so using spacing tokens wouldn't be appropriate because changing the spacing tokens would ruin my fix. Instead I'll parse the shadow token and retrieve these values from there.